assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -41,23 +47,51 @@ import {
   PulseAnimation,
   RotateInLeftAnimation,
   RotateInRightAnimation,
+  TwirlInAnimation,
+  WhooshInLeftAnimation,
+  WhooshInRightAnimation,
+  PanRightAnimation,
+  PanBottomAnimation,
+  PanTopAnimation,
+  PanLeftAnimation,
+  ZoomInAnimation,
+  ZoomOutAnimation,
+  BaseAnimationCell,
 } from './effectChooserElements';
 
 const Container = styled.div`
   background: black;
   width: ${PANEL_WIDTH}px;
 `;
 
-const GridItem = styled.button`
+const ContentWrapper = styled.div`
+  display: inline-block;
+`;
+
+const GridItem = styled.button.attrs({ role: 'listitem' })`
   border: none;
   background: #333;
   border-radius: 4px;
   height: ${GRID_ITEM_HEIGHT}px;
   position: relative;
   overflow: hidden;
+  font-family: 'Teko', sans-serif;
+  font-size: 20px;
+  color: white;
+  text-transform: uppercase;
+
+  &:hover {
+    ${BaseAnimationCell} {
+      display: inline-block;
+    }
+
+    ${ContentWrapper} {
+      display: none;
+    }
```

Comment:
it would be nice if this was also on `:focus` so that if you're using a keyboard you still get the preview